examples/src/main/java/com/expediagroup/sdk/lodgingconnectivity/SandboxDataManagementClientUsageExample.java

```diff
@@ -133,33 +123,24 @@ public static void main(String[] args) {
                 .build();
 
         ChangeSandboxReservationStayDatesResponse changeStayDatesResponse = client.changeReservationStayDates(changeReservationStayDatesInput);
-
         reservationData = changeStayDatesResponse.getData();
+        logger.info("Reservation Stay Dates Updated: {}", reservationData.getId());
 
-        System.out.println("Reservation Stay Dates Updated: " + reservationData.getId());
-        System.out.println(changeStayDatesResponse);
 
         // ******* Cancel Reservation *******
         CancelSandboxReservationResponse cancelReservationResponse = client.cancelReservation(reservationData.getId());
-
         reservationData = cancelReservationResponse.getData();
-
-        System.out.println("Reservation Was Canceled: " + reservationData.getId());
-        System.out.println(cancelReservationResponse);
+        logger.info("Reservation Was Canceled: {}", reservationData.getId());
```

Comment:
We usually log ids and digits in between square brackets [] can you update these all the logs in this file.

Reply:
Done